package.json

```diff
@@ -53,5 +53,8 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
+    },
+    "dependencies": {
+        "sync-storage": "^0.4.2"
```

Comment:
You shouldn't touch the root `package.json`, the package itself will automatically get installed in the CI flow.

Reply:
Sorry about this. I have fixed this issue.